data_tools/CHANGELOG.rst

```diff
@@ -0,0 +1,12 @@
+Changelog
+=========
+
+This document describes changes between each past release.
+
+0.1 (unreleased)
+----------------
+
+**Initial version**
+
+- Scraping tools from archive
+- Scraping tools for systemaddons
```

Comment:
Technically the systemaddons thing is not a scraper. So just write something short like:
* Scrape product releases and nightlies from https://archives.mozilla.org
* Fetch system addons update information from Balrog